layer/README.md

```diff
@@ -0,0 +1,26 @@
+<!-- markdownlint-disable MD041 MD043-->
+# CDK Powertools layer
+
+This is a CDK project to build and deploy AWS Lambda Powertools layer to multiple commercial regions.
+
+## Build the layer
+
+To build the layer construct you need to provide the powertools version.
+You can pass it as a context variable when running `synth` or `deploy`,
+
+```shell
+cdk synth --context version=1.25.1
+```
+
+## Canary stack
+
+We use a canary stack to verify that the deployment is successful and we can use the layer by adding it to a newly created Lambda function.
```

Comment:
```suggestion
We use a canary stack to verify that the deployment is successful and we can use the Layer by adding it to a newly created Lambda function.
```

Capital `L` for Layer.

Reply:
Why capital L 🤔? It's in lower case in the docs.

Reply:
we should be fixing those then (rewriting examples now, then branding and doc rewrite). The reason is that `Layer` with capital L means a reference to a product or something (copywriting), while `layer` being a noun.

Reply:
No I mean `layer` is used in the [official docs](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html)

Reply:
Ah! that's because they introduce it with `Lambda layer` and points to a glossary which we don't have. From there they continuously refer to Layer as it belongs to Lambda. Maybe we link to it and keep as `layer` then?

https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-concepts.html#gettingstarted-concepts-layer